cirq-core/cirq/devices/unconstrained_device.py

```diff
@@ -26,6 +26,7 @@
 class _UnconstrainedDevice(device.Device):
     """A device that allows everything, infinitely fast."""
```

Comment:
[No change needed] What exciting technology!

Reply:
We live in exciting times lol.